.nuke/parameters.ci-chrome.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "./build.schema.json",
+  "Solution": "Remotion.sln",
+  "Target": [ "CIBuild" ],
+  "TestConfigurations": [ "Debug", "Release" ],
+  "TestExecutionRuntimes": [ "EnforcedLocalMachine" ],
```

Comment:
browser build uses docker. the old script had 
```
      <ExecutionRuntimes>Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
```

Reply:
Okay maybe I misunderstood the hack with `EnforcedLocalMachine` but I thought that this setup runs the tests locally, but hosts the test site in the specified docker container. Otherwise `EnforceLocalMachine` is really bad naming.

The `WebTestSetup` class is used in the web tests to pass the required docker image name to the web tests. This is no longer hacked into test dimension matrix.